python/lsst/afw/geom/ellipses/ellipse.cc

```diff
@@ -22,24 +22,36 @@
 
 #include <pybind11/pybind11.h>
 //#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/stl.h>
+
+#include "lsst/afw/geom/ellipses/BaseCore.h"
+#include "lsst/afw/geom/Point.h"
+#include "lsst/afw/geom/ellipses/Ellipse.h"
 
 namespace py = pybind11;
+using namespace pybind11::literals;
 
-using namespace lsst::afw::geom::ellipses;
+using namespace lsst::afw::geom;
+using namespace ellipses;
```

Comment:
I'm still curious, but it's not worth doing here. I'll try it when it's my turn to do some pybind11 wrapping.